test/integration/main_test.go

```diff
@@ -314,6 +317,9 @@ func degradeOperatorMonitoring(t *testing.T) func() {
 func changeReportTimeInterval(t *testing.T, newInterval []byte) []byte {
 	supportSecret, _ := clientset.CoreV1().Secrets(OpenShiftConfig).Get(context.Background(), Support, metav1.GetOptions{})
 	previousInterval := supportSecret.Data["interval"]
+	if previousInterval == nil {
+		t.Fatal("missing secret with name", Support)
+	}
```

Comment:
Well `supportSecret` can be nil right so this should be updated IMO. We should also check the error and not ignore it IMO.

Reply:
+1